client/App.jsx

```diff
@@ -23,32 +29,68 @@ class App extends React.Component {
       method: 'GET',
       success: (response) => {
         this.setState({
+          restaurantId,
           restaurantInfo: response,
         });
       },
     });
   }
 
+  closeModal() {
+    this.setState({
+      showResults: false,
+    });
+  }
+
+  submitSearch(modal) {
+    const { restaurantId } = this.state;
+    const date = document.getElementById(`DateSelect${modal ? 'Modal' : ''}`).value;
+    const time = document.getElementById(`TimeSelect${modal ? 'Modal' : ''}`).value;
+    const people = document.getElementById(`PeopleSelect${modal ? 'Modal' : ''}`).value;
+
+    $.ajax({
+      url: `/api${restaurantId}search`,
```

Comment:
Do you need to include the slashes to match your route /api/:id/search?

Reply:
when i assign restaurantId with `window.location.pathname` , it includes the slashes, so i do not include them again in the template literal for the url